frontend/src/composables/credential/useCloudProviderBinding.js

```diff
@@ -22,44 +34,103 @@ export const useCloudProviderBinding = (binding, options = {}) => {
   if (!isRef(binding)) {
     throw new TypeError('First argument `binding` must be a ref object')
   }
+
   const {
     shootStore = useShootStore(),
+    cloudProfileStore = useCloudProfileStore(),
+    gardenerExtensionStore = useGardenerExtensionStore(),
+    credentialStore = useCredentialStore(),
   } = options
   const shootList = shootStore.shootList
 
-  const isSharedCredential = computed(() => {
-    return _isSharedCredential(binding.value)
+  // Classification Flags
+  const isSharedCredential = computed(() =>
+    _isSharedCredential(binding.value),
```

Comment:
to not have this naming conflict we could suffix the function with `Value`

export function isSharedCredentialValue(binding) {
  return /* boolean */
}

Reply:
let's leave it for now